components/core/tests/test-ir_encoding_methods.cpp

```diff
@@ -0,0 +1,340 @@
+// Catch2
+#include "../submodules/Catch2/single_include/catch2/catch.hpp"
+
+// Project headers
+#include "../src/ffi/encoding_methods.hpp"
+#include "../src/ffi/ir_stream/encoding_methods.hpp"
+#include "../src/ffi/ir_stream/decoding_methods.hpp"
+#include "../src/ffi/ir_stream/protocol_constants.hpp"
+
+using ffi::decode_float_var;
+using ffi::decode_integer_var;
+using ffi::decode_message;
+using ffi::eight_byte_encoded_variable_t;
+using ffi::four_byte_encoded_variable_t;
+using ffi::encode_float_string;
+using ffi::encode_integer_string;
+using ffi::encode_message;
+using ffi::get_bounds_of_next_var;
+using ffi::VariablePlaceholder;
+using ffi::wildcard_query_matches_any_encoded_var;
+using std::string;
+using std::vector;
+using ffi::ir_stream::IRErrorCode;
+using std::chrono::milliseconds;
+using std::chrono::system_clock;
+
```

Comment:
Let's add some more `using`s to reduce the amount of repetition.

Reply:
fixed